community/rpm.sh

```diff
@@ -44,8 +44,8 @@ pgdg_check ()
 
     yum install -d0 -e0 -y "${repo_url}"
 
-    if ! yum info -q -y postgresql15-server &> /dev/null; then
-      echo "PGDG repositories don't have postgresql15-server package for your operating system"
+    if ! yum info -q -y postgresql16-server &> /dev/null; then
+      echo "PGDG repositories don't have postgresql16-server package for your operating system"
```

Comment:
This conditional clause can cause failures in el/7 where postgresql-16 packages are not supported.